GUI/Labels/ModuleLabel.cs

```diff
@@ -42,6 +42,18 @@ public class ModuleLabel
         [JsonProperty("module_identifiers", NullValueHandling = NullValueHandling.Ignore)]
         public HashSet<string> ModuleIdentifiers = new HashSet<string>();
 
+        /// <summary>
+        /// Check whether this label is active for a given game instance
+        /// </summary>
+        /// <param name="instanceName">Name of the instance</param>
+        /// <returns>
+        /// True if active, false otherwise
+        /// </returns>
+        public bool AppliesTo(string instanceName)
```

Comment:
I wouldn't want to implement a get/set property that returns different values than I set for it, and I'm also not happy with the idea of referencing a global variable here, but I'll take a look at whether we can simplify some of this with some changes in the list class...

Reply:
Created a `ModuleLabelsList.LabelsFor(instanceName)` function because it's common logic in so many places, but I think passing the instance name is appropriate given what we're calculating.

Reply:
Okay, at least removes some of the duplicate code and makes it easier to read :+1: